Cohort2/team4/server/src/models/User.ts

```diff
@@ -1,7 +1,7 @@
 import { Table, Column, PrimaryKey, Model, Unique, AutoIncrement, DataType, AllowNull } from 'sequelize-typescript';
 
-@Table({ tableName: 'user' })
-class User extends Model {
+@Table({ tableName: 'users' })
```

Comment:
Same here. Set timestamps to false in the @Table tag, after the tableName property.